build.sbt

```diff
@@ -21,7 +21,12 @@ sonatypeProfileName := "org.virtuslab"
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
-ThisBuild / semanticdbEnabled := true
+ThisBuild / semanticdbEnabled := {
+  scalaVersion.value match {
+    case "2.13.7" | "2.13.8" => false
+    case _                   => true
+  }
+}
 ThisBuild / semanticdbVersion := scalafixSemanticdb.revision
```

Comment:
```suggestion
ThisBuild / semanticdbEnabled := true
ThisBuild / semanticdbVersion := {
  scalaVersion.value match {
    case "2.13.7" | "2.13.8" => "4.4.32"
    case _                   => "4.4.28"
  }
}
```

Reply:
Seems to do the job for me locally

Reply:
Yes. It works now on all versions and CI jobs pass also.